package.json

```diff
@@ -66,7 +66,10 @@
     "yargs": "^17.0.1"
   },
   "devDependencies": {
+    "@types/electron-packager": "^15.0.1",
+    "@types/hasbin": "^1.2.0",
     "@types/jest": "^26.0.23",
+    "@types/loglevel": "^1.6.3",
```

Comment:
Same question as above: loglevel ships its own types since https://github.com/pimterry/loglevel/releases/tag/v1.6.2 . Do you have a reason to use DT types?

> Include TypeScript type definitions in the package itself

Reply:
Yeah. It's weird, it says that, but was also getting some warning here and there that it couldn't find types. So 🤷